stable/mlrun/templates/db-deployment.yaml

```diff
@@ -146,13 +146,19 @@ spec:
         podAffinity:
           {{- if eq .Values.db.defaultAffinityType "preferred" }}
           preferredDuringSchedulingIgnoredDuringExecution:
+            - weight: 100
+              podAffinityTerm:
+                labelSelector:
+                  matchLabels:
+                    {{- include "mlrun.db.selectorLabels" . | nindent 18 }}
+                topologyKey: "kubernetes.io/hostname"
           {{- else }}
           requiredDuringSchedulingIgnoredDuringExecution:
-          {{- end }}
             - labelSelector:
                 matchLabels:
-                  {{- include "mlrun.api.selectorLabels" . | nindent 18 }}
+                  {{- include "mlrun.db.selectorLabels" . | nindent 16 }}
```

Comment:
I think it was api selector labels on purpose

Reply:
Same, the DB pod will be looking for another DB pod to co-locate with this change